scripts/qa_crowbarsetup.sh

```diff
@@ -2531,7 +2531,7 @@ function adapt_dns_for_docker()
 function oncontroller_testsetup()
 {
     . .openrc
-
+    [[ $want_mtu_size ]] && safely ping -M do -c 1 -s $(( want_mtu_size - 28 )) $adminip
```

Comment:
storage and sdn are harder to test, because they might not be setup with larger MTU yet and we dont know a suitable address to ping